torch_xla/csrc/aten_xla_bridge.cpp

```diff
@@ -39,6 +39,16 @@ class AtenXlaDeviceMapper {
     return devices_;
   }
 
+  void SetVirtualDevice() {
```

Comment:
Per our offline discussion, with @JackCaoG , we will re-init the mapping so that it only contains the virtual device.

Reply:
Done, I will test if this can happen now, or after we make the device mutable. cc @JackCaoG

Reply:
should we remove this since it is not used?